Assets/ScaleMonk Ads/Source/AdsEditorBinding.cs

```diff
@@ -33,6 +35,25 @@ public MockAd CreateMockAdInstance()
             mockAdInstance.SetScalemonkAds(_scaleMonkAds);
             return mockAdInstance;
         }
+
+        public MockBannerAd CreateBannerMockAdInstance()
+        {
+            MockBannerAd mockAdInstance;
+            MockBannerAd mockAdPrefab;
+
+            if (isPortrait())
```

Comment:
`mockAdPrefab = Resources.Load<MockBannerAd>("Prefabs/MockAd_banner_" + isPortrait()? "portrait":"landscape");`

Reply:
it avoid repeated code and reduce a lot of lines

Reply:
Done!